Changelog.md

```diff
@@ -1,6 +1,7 @@
-## 0.4.0.dev (unreleased)
+## 1.0.0.skroutz.0 (unreleased)
 
- * ???
+ * Enable non forking resque workers when `fork: false` is present for a queue's workers
```

Comment:
I think `fork: false` is inconsistent wrt the code. It should be `fork_per_job: false`, correct?

Reply:
Ah yes! Thanks for that! Fixed!